# request
GraspCommand grasp_goal
---
#response
GraspCommand grasp_result
